lib/index.js

```diff
@@ -406,7 +406,10 @@ Title.newFromText = function(title, siteInfo, defaultNs) {
     }
 
     if (result.namespace.isSpecial()) {
-        result.title = _normalizeSpecialPage(result.title, siteInfo);
+        // This is mainly called in ApiPageSet::processTitlesArray() and
+        // is probably not strictly necessary, but doesn't hurt and Parsoid
+        // appreciates it.
```

Comment:
Hm.. I don't quite understand why is it not necessary? The goal of the library is to perfectly map mediawiki behavior. If mediawiki does that, it's nesessary in this lib as well..